src/test/java/uk/gov/hmcts/reform/professionalapi/sort/ProfessionalApiSortTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.professionalapi.sort;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+import uk.gov.hmcts.reform.professionalapi.domain.Organisation;
+import uk.gov.hmcts.reform.professionalapi.domain.ProfessionalUser;
+import uk.gov.hmcts.reform.professionalapi.domain.ProfessionalUserStatus;
+
+public class ProfessionalApiSortTest {
+
+    private final Organisation organisationMock = mock(Organisation.class);
+
+    @Test
+    public void shouldSortProfessionalUserList() {
+
+        ProfessionalUser user1 = new ProfessionalUser("firstName1",
+                "lastName1",
+                "emailAddress1@org.com",
+                ProfessionalUserStatus.PENDING,
+                organisationMock);
+
+
+        ProfessionalUser user2 = new ProfessionalUser("firstName2",
+                "lastName2",
+                "emailAddress2@org.com",
+                ProfessionalUserStatus.PENDING,
+                organisationMock);
+
+        user1.setCreated(LocalDateTime.now());
+        try {
+            Thread.sleep(1000);

```

Comment:
Need to remove this in a follow-up Jira. We should not be intentionally slowing down the unit test suite as everyone now will have to wait an extra second whenever they wish to run their test suit. Doesn't sound like a lot but a team of developers running tests dozens of times a day means this will get annoying quickly. I suggest moving to a functional test if you have no other options but to put the thread to sleep. Here you could have used strings and converted them to LocalDateTime instances.